src/Support/helpers.php

```diff
@@ -54,3 +54,18 @@ function lad_pr_me($name, \Closure $action = null)
         return app(Debugger::class)->profileMe($name, $action);
     }
 }
+
+
+if (! function_exists('convert_to_unit')) {
```

Comment:
Nice feature, but move this helper to collection itself and make it a little bit more readable

Reply:
I have resolve it.

Reply:
Awesome! And one more, add some tests please